packages/caver-rpc/src/governance.js

```diff
@@ -161,13 +161,35 @@ const Governance = function Governance(...args) {
          * const result = await caver.rpc.governance.getChainConfig()
          *
          * @param {function} [callback] Optional callback, returns an error object as the first parameter and the result as the second.
-         * @return {Promise<object>} The initial chain configuration
+         * @return {Promise<object>} The latest chain configuration
          */
         new Method({
             name: 'getChainConfig',
             call: 'governance_chainConfig',
             params: 0,
         }),
+        /**
+         * Returns the chain configuration at specific block.
```

Comment:
There's a minor typo.
```suggestion
         * Returns the chain configuration at a specific block.
```